data/monitoring/prom-rule.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus.cnao.io: ""
+  name: prometheus-rules-cluster-network-addons-operator
+  namespace: {{ .Namespace }}
+spec:
+  groups:
+    - name: kubevirt.cnao.rules
+      rules:
+        - expr: sum(up{namespace='{{ .Namespace }}', pod=~'cluster-network-addons-operator-.*'} or vector(0))
+          record: kubevirt_cnao_num_up_operators
+        - alert: CnaoDown
+          annotations:
+            summary: CNAO pod is down.
+          expr: kubevirt_cnao_num_up_operators == 0
+          for: 5m
+          labels:
+            severity: critical
```

Comment:
@RamLavi  Does this alert severity fit the description?
Critical - When the service is down and you loss critical functionality, an action is required immediately

Reply:
if CNAO operator is down then there's no one to make sure that CNAO's components are running, one of them is Kubemacpool. if Kubemacpool is down, then VMs can't be created. 
so considering your definition, maybe warning is more fitting?

Reply:
changed to Warning. Done